### ISSUE TITLE
Please add support for the dreame x40 master

### ISSUE THREAD

**Issue by Schatn** [Tasshack/dreame-vacuum#786]
Please add support for the dreame x40 master, thank you

**Comment by Schatn** [comment]
Thanks for your very fast reply.
The first 5 chars of the serial number are R2465.

**Comment by EternalSentinel** [comment]
I understand that the X40 Master is already supported and I have one with serial beginning with R2465B and I received the message "No supported devices found in this account on selected country”. Does the last digit matter and if so, will it be supported in the future?

**Comment by Tasshack** [comment]
> I understand that the X40 Master is already supported and I have one with serial beginning with R2465B and I received the message "No supported devices found in this account on selected country”. Does the last digit matter and if so, will it be supported in the future?

Yes, `dreame.vacuum.r2465b` is not supported yet, can you create an issue about that.
